util/hatchet/_generate_hatchet_token.sh

```
@@ -0,0 +1,59 @@
+    echo ""
+    echo "Please start the services first by running:"
+    echo "   ./_run_app_dependencies_macos.sh"
+    echo ""
+    exit 1
+fi
+
+echo "✅ All required Hatchet services are running"
+
+# Run the token generation from the hatchet directory (so relative paths work)
+echo "🔑 Generating Hatchet client token..."
+cd "$SCRIPT_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)